examples/OpenTracing.Examples/ActiveSpanReplacement/ActiveSpanReplacementTest.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OpenTracing.Mock;
+using Xunit;
+
+using static OpenTracing.Examples.TestUtils;
+
+namespace OpenTracing.Examples.ActiveSpanReplacement
+{
+    public class ActiveSpanReplacementTest
+    {
+        private readonly MockTracer _tracer = new MockTracer();
+
+        [Fact]
+        public void test()
+        {
+            // Start an isolated task and query for its result in another task/thread
+            ISpan initialSpan = _tracer.BuildSpan("initial").Start();
```

Comment:
So the scenario to reproduce is 

1. To pass a `Span` to a task, **without** it being parent of further created `Span`s
2. Later on in the task, reactivate the passed `Span` and finish it.

Since context is propagated in C#, further created `Span`s would become children of it, and would have ended up using `IgnoreActiveSpan()` to have the same effect.

(I should do a PR against that Java example, do not activate the `Span` ;) )